less/availity-angular.less

```diff
@@ -10,3 +10,17 @@
   background: none;
   z-index: 1011;
 }
+
+.dimmer-content {
```

Comment:
Any reason we wouldn't just use bootstrap 4's card-img-overlay? http://v4-alpha.getbootstrap.com/components/card/#image-overlays

It has everything you need except the alignment and colors